src/components/panel/demoGroups/style.global.css

```diff
@@ -0,0 +1,80 @@
+.menu-panel-container {
+  pointer-events: auto;
+}
+
+.menu-panel {
+  width: 256px;
+  background-color: #ffffff;
+  border-radius: 4px;
+  box-shadow: 0px 1px 5px 0px rgba(0, 0, 0, 0.2),
+      0px 2px 2px 0px rgba(0, 0, 0, 0.14),
+      0px 3px 1px -2px rgba(0, 0, 0, 0.12);
```

Comment:
@DevVersion This is CSS. I am unsure how to use Sass variables without it being Sass.